roboteam_ai/src/world/World.cpp

```diff
@@ -66,12 +66,45 @@ void World::updateWorld(proto::World &protoWorld) {
     setWorld(data);
 }
 
+// Converts millimeters to meters
+double mmToM(double mm) {
```

Comment:
Will do! (I think this will be a temporary function anyways, as this is only used in the conversion between proto and c++ Fields, which hopefully moves to Observer)